doc/classes/Engine.xml

```diff
@@ -310,6 +310,9 @@
 		</method>
 	</methods>
 	<members>
+		<member name="fixed_fps" type="int" setter="set_fixed_fps" getter="get_fixed_fps" default="-1">
+			When fixed_fps is greater than [code]-1[/code], the engine will process at this fps regardless of what the actual fps is. This is useful for doing things such as recording movies where realtime updates do not matter, but the images captured should appear smooth once being encoded together.
```

Comment:
What happens if it's set to 0? Is this documented elsewhere where it can be set?

Reply:
I do not know.  I'll go set it and see what happens.  Hopefully the world doesn't end 😅

Reply:
Would be useful to mention!

Reply:
Unfortunately I appear to have ended the world... 😅 lol jk jk but it does behave strange.  It starts updating in unexpected ways.  And no longer allows me to set the fixed fps back to its previous value.  Setting it to a value of 1 works fine.  But 0 does not.  There's a few things I can do:

1. Prevent users from setting to 0.  Change to 1 if so
2. Prevent users from setting to 0.  Error and return.
3. Allow users to set to 0, but document the strange behavior / warn against it.

For all of those, I'll update the Engine.xml with whatever we decide.  What are your thoughts?  I'm thinking the simplest and most intuitive approach would probably be just error and return + document that in the xml.

ooooooooooorrrrrrrrrr there is a 4th option:

I can make it so any value <= 0 behaves like -1 currently behaves... I kinda like this solution now that I think about it.  This removes all potential errors and edge cases.  I believe its what @Calinou did in his original PR that @programneer mentioned here: https://github.com/godotengine/godot/pull/92490#issuecomment-2136789520

Funny how multiple people come to the same solution on a problem 😅.  This must be how convergent evolution works!  All the way until we all become crabs!  🦀 https://www.popsci.com/story/animals/why-everything-becomes-crab-meme-carcinization/

Reply:
Indeed didn't realize that this was a duplicate PR, think that one is more complete and should be merged instead, but we'll see what's decided

Reply:
I agree.  I just looked through it and it has better documentation.  I think we should do that PR instead.  Hey @Calinou !  I brought ur PR back to life! :)  On accident 😅.  I prefer your better docs compared to mine so I think I'll merge your branch into my fork :)